lib/gollum/views/editable.rb

```diff
@@ -4,6 +4,18 @@ def has_editor
       true
     end
 
+    def default_keybinding
+      @default_keybinding
+    end
+
+    def keybindings
+      ["default", "vim", "emacs"].map do |kb|
```

Comment:
Single quotes it is :)

I chose to hard code them at the exact and only place the entire list was needed. 
 
The list of supported keybinds can be defined in `lib/gollum.rb`. This way it may be referenced through `Gollum::`  namespace. I originally chose not to define it in `lib/gollum.rb` because the code here looks intentionally minimal. In my opinion, `lib/gollum.rb` did not seem to be the place to define supported constant values for a minor configuration option. 

The most recent commit demonstrates this approach. I'm happy to remove it if decided.

Reply:
> The most recent commit demonstrates this approach.

Looks good!